static/js/containers/pages/b2b/B2BReceiptPage.js

```diff
@@ -130,36 +137,68 @@ export class B2BReceiptPage extends React.Component<Props, State> {
               <div className="title">Bulk Seats Receipt</div>
             </div>
           </div>
-          <div className="row">
+          <div className="row purchase-summary-block">
             <div className="col-lg-8">
               <p>
                 Thank you! You have purchased one or more seats for your team.
               </p>
-              <h3>Purchase Summary (Order Number):</h3>
-              <p className="course-or-program">
-                <span className="description">Course or program:</span>
-                {title}
-                <span className="description">{readableId}</span>
-              </p>
-              <p className="seats">
-                <span className="description">Seats:</span>
-                {numSeats} (at {formatPrice(itemPrice)} per seat)
-              </p>
-              <p className="email">
-                <span className="description">Email Address:</span>
-                {email}
-              </p>
-              {contractNumber && (
-                <p className="contract-number">
-                  <span className="description">Contract Number:</span>
-                  {contractNumber}
+              <B2BReceiptExplanation className="b2b-receipt-explanation" />
+              <div className="text-box">
+                <h4>
+                  Purchase Summary ({referenceNumber ? referenceNumber : null}):
+                </h4>
+                <p className="order-date">
+                  <span className="description">Order Date:</span>
+                  {moment(createdOn).format("MMMM DD, YYYY")}
```

Comment:
Please use `formatPrettyDate` in `static/js/lib/util.js` to format this

Reply:
but strange, this pattern is also in code base.